src/components/Proposal/ProposalData.tsx

```diff
@@ -163,7 +163,7 @@ export default withSubscription({
         // TODO: also need the member state for the proposal proposer and beneficiary
         //      but since we need the proposal state first to get those addresses we will need to
         //      update the client query to load them inline
-        concat(of(new BN("0")), await ethBalance(daoState.address)
+        concat(of(new BN("0")), (await ethBalanceDao(daoState.address))
```

Comment:
why proposalData page need to pull for the dao balance ?

Reply:
the question is out of scope of this PR, but the answer is to feed the `ActionButton` component which uses it for the redeem button.

Reply:
we poll in order to keep the value up-to-date

Reply:
I think it can avoided here.
When and if the use press the redeem button it should read the balance.

Reply:
> I think it can avoided here.
> When and if the use press the redeem button it should read the balance.

That is not within the scope of this PR